common/api/summary/undeprecate-3.exports.csv

```diff
@@ -0,0 +1,15 @@
+sep=;
+Release Tag;API Item
```

Comment:
Looks like this was accidentally added?

Reply:
I have no idea what that means. I assume the extract-api did that?

Reply:
I was talking about the entire file. The extract-api summary files should only be per-package so not user where `undeprecate-3` comes from. I'd just delete the file.

Reply:
Done.